index.html

```diff
@@ -1,99 +1,95 @@
+                    </svg>
```

Comment:
- [ ] I am sorry that you had to go through this stress because of the misconception of what the previous reviewer meant. He was asking you to please use a `a` tag as in an anchor tag (`<a href="index.html">Your Logo Text Here </a>` so that users or visitors can always access your homepage from wherever they are on the page when they click on the logo. :smile: This improves site's accessibility and gives users quality user experience. Now that you have used a `svg` tag in place of texts, kindly wrap the `svg` content in a `a` tag so that users can have blissful experience on your page. By doing this, you will also be implementing one of HTML's best practices :hugs:

Reply:
Thank you for the explanation. I finally understand what he wanted.

- [x] wrap the svg content in a a tag so that users can have blissful experience on your page